documentation/modules/ROOT/partials/modules/mysql-connector/ref-mysql-connector-configuration-properties.adoc

```diff
@@ -282,6 +282,8 @@ _Note:_ This feature should be considered an incubating one. We need a feedback
 
 `minimal` The connector holds the global read lock for just the initial portion of the snapshot while the connector reads the database schemas and other metadata. The remaining work in a snapshot involves selecting all rows from each table, and this can be done in a consistent fashion using the REPEATABLE READ transaction even when the global read lock is no longer held and while other MySQL clients are updating the database. +
 
+`minimal_percona` The connector holds the global backup lock for just the initial portion of the snapshot while the connector reads the database schemas and other metadata. The remaining work in a snapshot involves selecting all rows from each table, and this can be done in a consistent fashion using the REPEATABLE READ transaction even when the global read lock is no longer held and while other MySQL clients are updating the database. This mode does not flush tables to disk and is not blocked by long-running reads, and is only available in Percona Server +
```

Comment:
Is there any description in the Percona docs where "global backup lock" could link to?